### ISSUE TITLE
set ttl for a s3 snapshot

### ISSUE THREAD

**Issue by botzill** [kubedb/project#440]
Hi.

 Is there a way to set automatically a TTL for S3 snapshot? Or a way to say "keep latest x snapshots"? Would be great to have such a feature automatically.

Thx!

**Comment by calleo** [comment]
I guess you could do this directly on the s3 bucket as well?

**Comment by botzill** [comment]
Yes, we can do that as well, but this requires extra effort, especially when you want to automate things Having smth that works out of the box would be great. We just install the DB and it works :).


Thx!
